packages/notifi-axios-adapter/lib/mutations/createSupportImpl.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  collectDependencies,
+  makeParameterLessRequest,
+} from '@notifi-network/notifi-axios-utils';
+import { CreateSupportResult } from '@notifi-network/notifi-core';
+
+const DEPENDENCIES: string[] = [];
+
+const MUTATION = `
+mutation createSupport {
```

Comment:
Nit: Could we update the mutation name to `createSupportConversation` and cross the function / file names? 

It would help with readability and understanding what the implementation does, and most, if not, mutations in the `axios-adapter` follows this convention. It should work because `mutation` whatevername is the name of the function that assess the `createSupportConversation` mutation in the backend.

If you are under a time crunch, no worries, keep it like this and I can approve.

Reply:
updated the function name in useNotifiClient from createSupport to createSupportConversation

Reply:
Hey Hao, thank you for the update! I was wondering if you could do it for the components AND the mutation name as well. I'll approve because it is a nit but you can get back to it when you can.

Reply:
Hi Jennifer, thanks for pointing out, I have updated the mutation name and all the name from createSupport to createdSupportConversation.